src/objective-c/GRPCClient/GRPCTypes.h

```diff
@@ -23,7 +23,7 @@
  * Note that a few of these are never produced by the gRPC libraries, but are of
  * general utility for server applications to produce.
  */
-typedef NS_ENUM(NSUInteger, GRPCErrorCode) {
```

Comment:
Thanks Daniel! LGTM in general :)

2 thoughts:

1. Is this a non-breaking change for all clients?
2. There are a few other enums in this file, as well as throughout the codebase, which i wonder would benefit from the same improvement?

Thanks :)